src/main/java/backend/UpdateController.java

```diff
@@ -22,7 +22,7 @@
 
 /**
  * Manages the flow of logic during a data retrieval cycle from the repository source.
- *
+ * <p>
```

Comment:
They are generated by IntelliJ's code formatting. 
Empty lines in header comments does not generate new lines in JavaDoc. No matter how many empty lines you have, the generated JavaDoc will just have one continuous line. Adding `<p>` is like adding a line break. IntelliJ assumes that you wanna have a line break when you have empty lines in the header comment, so it automatically adds `<p>` when doing code formatting. This can be turned off though, but I think it is good.